tensorflow_datasets/core/dataset_builder.py

```diff
@@ -1205,8 +1301,11 @@ def _download_and_prepare(self, dl_manager, download_config):
     logging.info("Updating split info...")
     self.info.update_splits_if_different(split_dict)
 
-  def _prepare_split(self, split_generator, pipeline):
-    beam = lazy_imports_lib.lazy_imports.apache_beam
+  # pylint: disable=used-before-assignment
+  def _prepare_split(self,
+                     split_generator: splits_lib.SplitGenerator,
+                     pipeline: beam.Pipeline) -> None:
```

Comment:
As per https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING, you should wrap all beam annotations inside `'beam.Xyz'` otherwise, it will crash for users which haven't installed Beam.

`pipeline: "beam.Pipeline")`

Reply:
Done